app/src/components/governance/PageProposal.vue

```diff
@@ -1,168 +1,41 @@
 <template>
-  <TmPage
-    data-title="Proposal"
-    class="readable-width"
-    :loading="$apollo.queries.proposal.loading"
-  >
+  <TmPage data-title="Proposal" :loading="$apollo.queries.proposal.loading">
     <TmDataNotFound v-if="!found" />
-    <template v-else>
-      <div class="proposal">
-        <div class="page-profile__header__info">
-          <span :class="proposal.status | lowerCase" class="proposal-status">
-            {{ status.badge }}
-          </span>
-          <div class="proposal-title__row">
-            <h2 class="proposal-title">{{ proposal.title }}</h2>
-          </div>
-        </div>
-
-        <div class="proposer-row">
-          <p class="proposer">
-            <template v-if="proposal.validator">
-              Proposed by {{ proposal.validator.name }}:
-              <Address :address="proposal.proposer.address" />
-            </template>
-            <template v-else-if="proposal.proposer">
-              Proposed by
-              <Address
-                :address="proposal.proposer.name || proposal.proposer.address"
-              />
-            </template>
-            <template v-else>
-              Unknown proposer
-            </template>
-          </p>
-          <div class="button-container">
-            <TmBtn
-              v-if="proposal.status !== 'Passed'"
-              id="deposit-btn"
-              value="Deposit"
-              :disabled="proposal.status !== 'DepositPeriod'"
-              color="primary"
-              @click.native="onDeposit"
-            />
-            <TmBtn
-              id="vote-btn"
-              value="Vote"
-              :disabled="
-                proposal.status !== 'VotingPeriod' ||
-                proposal.type === 'TREASURY'
-              "
-              color="primary"
-              @click.native="() => onVote()"
-            />
-          </div>
-        </div>
-      </div>
+    <div v-else class="proposal">
+      <ProposalHeader
+        :title="proposal.title"
+        :type="proposal.type"
+        :proposer="proposal.proposer.address"
+        :status="status"
+        @open-vote-modal="onVote"
+        @open-deposit-modal="onDeposit"
+      />
 
-      <TextBlock :content="proposal.description" />
+      <ProposalStatusBar
+        v-if="tallyHasValues"
+        :status="status.badge"
+        :status-begin-time="new Date(proposal.statusBeginTime).toUTCString()"
```

Comment:
why are we converting to an UTC here? this is an input

Reply:
it was ugly. But yeah, I can convert with moment or with UTC in the component itself

Reply:
Removed